docs/dev/bundle_upload.rst

```diff
@@ -0,0 +1,48 @@
+.. _bundle-upload-full-datasets:
+
+Bundle Upload for Full Datasets
+===============================
+
+The goal is to upload a new synthetic dataset for the whole world.
+These synthetic datasets are constructed so that we have a known
+true answer (called an oracle) for figuring out how to fit.
+
+Most people are uploading a bunch of data and the maybe going back and
```

Comment:
Move this paragraph and the one below to where the sentences following the one on line 48 are, and throw those sentences out since they are a shorter version of this.